src/aat/resources/features/F-057/F-057.feature

```diff
@@ -0,0 +1,64 @@
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [does not provide the valid sort direction in input parameters]
+    And it is submitted to call the [Get the pagination metadata for a case data search for Citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [has the 400 return code]
+    And the response has all the details as expected
+
+  @S-138
+  Scenario: must return 4xx response when request does not provide valid authentication credentials
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [does not provide valid authentication credentials]
+    Given it is submitted to call the [Get the pagination metadata for a case data search for Citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [has the 403 return code]
+    And the response has all the details as expected
+
+  @S-139
+  Scenario: must return negative response when request provides authentic credentials without authorized access to the operation
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [does not provide valid authorized access to the operation]
+    Given it is submitted to call the [Get the pagination metadata for a case data search for Citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [has the 403 return code]
+    And the response has all the details as expected
+
+  @S-140
+  Scenario: must return pagination metadata successfully for correct inputs
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    Given it is submitted to call the [Get the pagination metadata for a case data search for Citizen] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [has the 200 return code]
+    And the response has all the details as expected
+
+  @S-521
+  Scenario: must return 400 when casefields do not start with “case.”
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [does not provide valid casefields which starts with “case.”]
+    Given it is submitted to call the [Get the pagination metadata for a case data search for Citizen] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [has the 400 return code]
+    And the response has all the details as expected
+
+  @S-522
+  Scenario: must return 400 when security classification in input parameters is present and invalid
+    Given a user with [a detailed profile in CCD]
+    When a request is prepared with appropriate values
+    #And the request [does not provide valid security classification in input parameters]
```

Comment:
Please activate this line in all occurrences. We will need to get the relevant code merged from DSL-2 implementation.